### ISSUE TITLE
Comcast Work Page: Community filter issue

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/community-app#4188]
1. On clicking on work from the nav, open opportunities on the platform  filtered for Comcast work is displayed.
<img width="1393" alt="Screenshot 2020-04-07 at 1 58 13 PM" src="https://user-images.githubusercontent.com/58783823/78647378-252a0000-78d8-11ea-8778-256f16edba9b.png">

2. Work: Auto redirect to open opportunities on the platform filtered for Comcast work
The sub communities filter is not working in production (https://github.com/topcoder-platform/community-app/issues/3278).
This will be fixed as part of the V5 Challenge API migration (https://github.com/topcoder-platform/community-app/issues/4123)
So if we filter by Comcast community no challenges will be shown. Suggestion: We can display all challenges with no filter and apply the filter once the fix is in place.

**Comment by asandlin** [comment]
3. Learn: can this open the Thrive page in another window? it opens in the same window now but there's no way to get back to the comcast portal other than using the back button or typing the address in again

**Comment by Oanh-and-only-Oanh** [comment]
@asandlin, created a separate ticket for you issue.  https://github.com/topcoder-platform/community-app/issues/4192

**Comment by SathyaJayabal** [comment]
#2 is working as we are able to filter by comcast. Reducing the priority as only #1 is present
